nexus/src/app/sagas/instance_create.rs

```diff
@@ -243,176 +217,145 @@ async fn sic_alloc_server(
         .map_err(ActionError::action_failed)
 }
 
-async fn sic_allocate_network_interface_ids(
+/// Create a network interface for an instance, using the parameters at index
+/// `iface_index`, returning the UUID for the NIC (or None).
```

Comment:
`iface_index` in the comment, `nic_index` as the argument name?

(whatever we pick should probably be consistent with `sic_create_network_interface_undo` below)